lib/bolt/plugin.rb

```diff
@@ -143,11 +143,12 @@ def initialize(config, pal, analytics)
       @analytics = analytics
       @plugin_context = PluginContext.new(config, pal)
       @plugins = {}
+      @pal = pal
```

Comment:
Maybe i'm reading this wrong but it looks like we meant to have `pal` be an attribute anyway https://github.com/puppetlabs/bolt/pull/1301/files#diff-e55bc985f91060535fd013d69f316af6R139